x-pack/test/security_solution_cypress/config.ts

```diff
@@ -52,6 +52,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
           'riskyHostsEnabled',
           'riskyUsersEnabled',
           'insightsRelatedAlertsByProcessAncestry',
+          'threatIntelligenceEnabled',
```

Comment:
Is there any plan of adding specific TI plugin tests on Security Solution Cypress tests?

Reply:
@MadameSheema we have some e2e Cypress tests [in our plugin directly](https://github.com/elastic/kibana/blob/main/x-pack/plugins/threat_intelligence/cypress/integration/sources/indicators.spec.ts) and [this PR](https://github.com/elastic/kibana/pull/137954) updates [some e2e Cypress tests](https://github.com/elastic/kibana/pull/137954/files#diff-2cb85ba5d1effa234d5f55e4d4bfe1853ff6f97478ab0f36fbe95c9bb3f6c6b1R147) within the Security Solution plugin.

I wasn't thinking about adding any other tests, especially because this feature flag is temporary and _should_ be very short lived. We're planning on releasing everything in 8.5 (plans could change though).

Did you have anything specify in mind you wanted me/us to add?